brambling/forms/orders.py

```diff
@@ -531,8 +531,10 @@ def clean_email(self):
             raise ValidationError("You can't transfer an item to yourself!")
         return email
 
-    def clean_boughtitem(self):
+    def clean_bought_item(self):
         bought_item = self.cleaned_data['bought_item']
+        if not bought_item.can_transfer():
```

Comment:
We should also in `__init__` be setting the queryset for this to just be items that can transfer, perhaps?